tutordiscovery/templates/discovery/build/discovery/Dockerfile

```diff
@@ -12,13 +12,8 @@ RUN mkdir -p /openedx/discovery && \
     git clone $DISCOVERY_REPOSITORY --branch $DISCOVERY_VERSION --depth 1 /openedx/discovery
 WORKDIR /openedx/discovery
 
-# Fix TypeError at /jsi18n/
-# https://github.com/edx/course-discovery/pull/2860
-RUN curl https://github.com/edx/course-discovery/commit/cbbefd56a4efa67070288e94a260c5e069d035d6.patch | git apply -
-
-# Setup minimal yml config file, which is required by production settings
-COPY config.yml /openedx/config.yml
-ENV DISCOVERY_CFG /openedx/config.yml
+# Setup empty yml config file, which is required by production settings
+ECHO "{}" > /openedx/config.yml
```

Comment:
This causes an error on my local installation. It should be instead: `RUN echo ...`

Reply:
Also, I think we still need to define the `DISCOVERY_CFG` environment variable. Otherwise we face an error during init:

```
Creating tutor_local_discovery-job_run ... done     
python manage.py migrate --noinput                
Traceback (most recent call last):            
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)                                                                  
  File "/openedx/venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()                               
  File "/openedx/venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 325, in execute
    settings.INSTALLED_APPS            
  File "/openedx/venv/lib/python3.8/site-packages/django/conf/__init__.py", line 79, in __getattr__
    self._setup(name)                               
  File "/openedx/venv/lib/python3.8/site-packages/django/conf/__init__.py", line 66, in _setup
    self._wrapped = Settings(settings_module)
  File "/openedx/venv/lib/python3.8/site-packages/django/conf/__init__.py", line 157, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)                    
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import                                                                                                                                                  
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/openedx/discovery/course_discovery/settings/tutor/production.py", line 1, in <module>
    from ..production import *
  File "/openedx/discovery/course_discovery/settings/production.py", line 26, in <module>
    CONFIG_FILE = environ['DISCOVERY_CFG']
  File "/usr/lib/python3.8/os.py", line 675, in __getitem__
    raise KeyError(key) from None
KeyError: 'DISCOVERY_CFG'                           
make: *** [Makefile:72: migrate] Error 1
Connection to lilac.overhang.io closed.
```

Reply:
updated.